tests/test_buffer_property.py

```diff
@@ -0,0 +1,38 @@
+"Contains a collection of pytest tests that validates the buffer property."
```

Comment:
Is this intended to be a comment?

Reply:
It is expected as per `ni-style-guide` to define what a python file does at the start of the file.